ingestr/src/stripe_analytics/README.md

```diff
@@ -0,0 +1,60 @@
+# Stripe
```

Comment:
we'd like to remove this README file from here.

Reply:
ok